docs/specs/meta-clients/web3wallet/sdk-api.md

```diff
@@ -7,6 +7,9 @@ class Web3Wallet {
   
   // establish pairing from URI (BOTH)
   public abstract pair(params: { uri: string }): Promise<void>;
+
+  // for responder to set supported and special namespaces
+  public abstract setNamespaceConfig(params: { namespaceConfig: NamespaceConfig }): Promise<void>;
```

Comment:
```suggestion
  public abstract setNamespaceConfig(params: NamespaceConfig): Promise<void>;
```
this should be `params: NamespaceConfig` and params itself containing `supportedNamespaces & specialNamespaces` parameters